packages/effects-core/src/components/mesh-component.ts

```
@@ -0,0 +1,46 @@
+  override render (renderer: Renderer) {
+    if (renderer.renderingData.currentFrame.globalUniforms) {
+      renderer.setGlobalMatrix('effects_ObjectToWorld', this.transform.getWorldMatrix());
+    }
+    renderer.drawGeometry(this.geometry, this.material);
+  }
+
+  // TODO 点击测试后续抽象一个 Collider 组件
+  getHitTestParams = (force?: boolean): HitTestTriangleParams | void => {
+    const area = this.getBoundingBox();
+
+    if (area) {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)